packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -95,29 +95,23 @@ export default class BuildManifestPlugin {
           }
 
           const filesForEntry: string[] = []
-          for (const chunk of entrypoint.chunks) {
-            // If there's no name or no files
-            if (!chunk.name || !chunk.files) {
+
+          for (const file of entrypoint.getFiles()) {
```

Comment:
Could you add add something to the description about the difference between entrypoint.chunks and entrypoint.getFiles? I'm curious, and as ever the WebPack documentation is awful.